### PR TITLE
spiceai/datasets internal views

### PR DESCRIPTION
**Pull request by Jeadie**

### CODE REVIEW

File: .spice/datasets/eth_asset_transfers.yml [hunk 1]
```diff
@@ -0,0 +1,8 @@
+name: spiceai.datasets.eth_asset_transfers
```
Comment (phillipleblanc) [inline .spice/datasets/eth_asset_transfers.yml]: See my comment in Teams

File: .spice/datasets/eth_beacon_slots_json.yml [hunk 2]
```diff
@@ -0,0 +1,8 @@
+name: spiceai.datasets.eth_beacon_slots_json
```
Comment (phillipleblanc) [inline .spice/datasets/eth_beacon_slots_json.yml]: See my comment in Teams

File: .spice/datasets/eth_beacon_states_json.yml [hunk 3]
```diff
@@ -0,0 +1,8 @@
+name: spiceai.datasets.eth_beacon_states_json
```
Comment (phillipleblanc) [inline .spice/datasets/eth_beacon_states_json.yml]: See my comment in Teams

File: .spice/datasets/eth_token_mints.yml [hunk 4]
```diff
@@ -0,0 +1,8 @@
+name: spiceai.datasets.eth_token_mints
```
Comment (phillipleblanc) [inline .spice/datasets/eth_token_mints.yml:1]: See my comment in Teams

File: .spice/datasets/eth_wallet_token_balances.yml [hunk 5]
```diff
@@ -0,0 +1,8 @@
+name: spiceai.datasets.eth_wallet_token_balances
```
Comment (phillipleblanc) [inline .spice/datasets/eth_wallet_token_balances.yml]: See my comment in Teams

File: .spice/datasets/goerli_beacon_slots_json.yml [hunk 6]
```diff
@@ -0,0 +1,8 @@
+name: spiceai.datasets.goerli_beacon_slots_json
```
Comment (phillipleblanc) [inline .spice/datasets/goerli_beacon_slots_json.yml]: See my comment in Teams

File: .spice/datasets/goerli_beacon_states_json.yml [hunk 7]
```diff
@@ -0,0 +1,8 @@
+name: spiceai.datasets.goerli_beacon_states_json
```
Comment (phillipleblanc) [inline .spice/datasets/goerli_beacon_states_json.yml]: See my comment in Teams

File: .spice/datasets/goerli_token_mints.yml [hunk 8]
```diff
@@ -0,0 +1,8 @@
+name: spiceai.datasets.goerli_token_mints
```
Comment (phillipleblanc) [inline .spice/datasets/goerli_token_mints.yml:1]: See my comment in Teams

File: .spice/datasets/goerli_token_transfers_with_tx.yml [hunk 9]
```diff
@@ -0,0 +1,8 @@
+name: spiceai.datasets.goerli_token_transfers_with_tx
```
Comment (phillipleblanc) [inline .spice/datasets/goerli_token_transfers_with_tx.yml]: See my comment in Teams

File: .spice/datasets/polygon_asset_transfers.yml [hunk 10]
```diff
@@ -0,0 +1,8 @@
+name: spiceai.datasets.polygon_asset_transfers
```
Comment (phillipleblanc) [inline .spice/datasets/polygon_asset_transfers.yml]: See my comment in Teams

File: .spice/datasets/queries_token_transfers_with_tx.yml [hunk 11]
```diff
@@ -0,0 +1,8 @@
+name: spiceai.datasets.queries_token_transfers_with_tx
```
Comment (phillipleblanc) [inline .spice/datasets/queries_token_transfers_with_tx.yml]: See my comment in Teams

File: .spice/datasets/polygon_token_mints.yml [hunk 12]
```diff
@@ -0,0 +1,8 @@
+name: spiceai.datasets.polygon_token_mints
```
Comment (phillipleblanc) [inline .spice/datasets/polygon_token_mints.yml:1]: See my comment in Teams

File: .spice/datasets/eth_token_mints.yml (hunk 4 above)
Comment (Jeadie) [reply .spice/datasets/eth_token_mints.yml:1]: eth.token_mints is currently
```sql
    SELECT * FROM internal.eth.postgres.token_mints
    UNION ALL
    SELECT * FROM internal.eth.datalake.token_mints
    ```

**Review by phillipleblanc — APPROVED** [review]

**Pull request merged**